pkg/ipamd/datastore/data_store.go

```diff
@@ -1166,6 +1193,8 @@ func (ds *DataStore) UnassignPodIPAddress(ipamKey IPAMKey) (e *ENI, ip string, d
 	ipsPerCidr.With(prometheus.Labels{"cidr": availableCidr.Cidr.String()}).Dec()
 	ds.log.Infof("UnassignPodIPAddress: sandbox %s's ipAddr %s, DeviceNumber %d",
 		ipamKey, addr.Address, eni.DeviceNumber)
+	// Decrement ENI Utilization gauge when pod IP is removed
```

Comment:
nit: "Decrement ENI Utilization gauge when ENI is removed"

Reply:
This is when a pod is removed, right? Would I want something like Decrement ENI IP usage on pod deallocation

Reply:
Oh sorry, I was looking at the wrong code, you are right. Please ignore this comment